packages/@okta/i18n/src/properties/login.properties

```diff
@@ -742,7 +742,8 @@ oie.phone.alternate.title = your phone
 ## Email
 oie.email.authenticator.description = Verify with a link or code sent to your email
 oie.email.mfa.title = Verify with your email
-oie.email.verify.sentText = An email was sent to
+oie.email.verify.sentText = Check <$1>{0}</$1> for a verification message. Click the verification button in your email or enter the code below to continue.
+oie.email.verify.alternate.sentText = Check your email for a verification message. Click the verification button in your email or enter the code below to continue.
```

Comment:
Two questions -
* Do we also need to change the page title to `Verify with my email` to conform to the screenshot on this ticket - https://oktainc.atlassian.net/browse/OKTA-343133
* Does it make sense to change the message to `Check your email for a message with your code. Enter the code below.` (as per the screenshot in the ticket OKTA-343133). That way, when OKTA-343133 is done, there will only be an intermediate screen that conveys the user to either click on the link or enter the code on the next screen after clicking on a link.

Reply:
Asking @vincentvoong-okta about 1.
About 2: The new JIRA to improve email magic link UX will be done post EA. So I think it's better to change the text for `sentText` until then right?

Reply:
1. @vincentvoong-okta said it should be `Verify with your email`